coderd/util/slice/slice.go

```diff
@@ -107,3 +107,34 @@ func Ascending[T constraints.Ordered](a, b T) int {
 func Descending[T constraints.Ordered](a, b T) int {
 	return -Ascending[T](a, b)
 }
+
+// SymmetricDifference returns the elements that need to be added and removed
+// to get from set 'a' to set 'b'.
+// In classical set theory notation, SymmetricDifference returns
+// all elements of {add} and {remove} together. It is more useful to
+// return them as their own slices.
+// Example:
```

Comment:
You could also provide a testable example, see here: https://go.dev/blog/examples
See also: https://go.dev/src/sort/example_test.go